style.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+  list-style: none;
+  font-family: 'Roboto', sans-serif;
+}
+
+.logo {
+  text-decoration: none;
+}
+
+.header {
+  background-image: url(./Top-circles.png);
+  background-repeat: no-repeat;
+  background-color: #191765;
+  height: 100%;
+  padding: 0 40px 40px 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.nav a {
+  color: #fff;
+}
+
+.header h1 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 35px;
+  line-height: 72px;
+  color: #7d73fc;
+  padding: 63px 0;
+}
```

Comment:
- [ ] Great job styling this header. Unfortunately, it is still a bit different from the Figma design. At the moment, your `font-weight` attribute is not working properly. This is happening because when you imported the font, you only got the `font-weight: 500`. 
- [ ] To match the original design better, you should also consider removing the `padding-bottom`. It is currently too big when compared with the given design.

Did you know that you can click on the elements and they will provide you with the exact typography? This is what this element should look like:
![image](https://user-images.githubusercontent.com/66967274/148458065-08a0b603-91a3-431d-9001-2fa3ab86843a.png)